### PR TITLE
MODELIX-608 Health check for warming up the MPS cache

### PR DESCRIPTION
**Pull request by slisson**
New health check `loadModels` implemented. Multiple `moduleNamespace` parameters can be provided to specify the relevant modules.

### CODE REVIEW

**Review by odzhychko — DISMISSED** [review]

File: docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc [hunk 1]
```diff
@@ -30,3 +30,9 @@ Reports an unhealthy system whenever no project is loaded.
 
 Reports an unhealthy system when no virtual folders are available.
 This might also be true in case a project without virtual folders is fully loaded.
+
+=== loadModels
+
+Can be used to eagerly load model data avoiding a slow first query.
+Multiple `moduleNamespace` parameters can be provided to specify the relevant modules.
+All modules starting with the string provided by that parameter are included.
```
Comment (languitar) [inline docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc:38]: So an appropriate request would be?

```
/health?loadModels=true&moduleNamespace=foo.bar
```

A bit weird that parameters are now used for two purposes. One time for selecting the check, and then here as a special case for the modules.

In any case, a valid example call given this complexity would be good to have in the docs.

Comment (slisson) [reply docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc:38]: I solved the part that requires MPS knowledge. Feel free to continue working on this PR.

Comment (odzhychko) [reply docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc:38]: Ok. Thank you for solving the MPS parts. I can continue with the details.

File: docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc [hunk 2]
```diff
@@ -30,3 +42,25 @@ Reports an unhealthy system whenever no project is loaded.
 
 Reports an unhealthy system when no virtual folders are available.
 This might also be true in case a project without virtual folders is fully loaded.
+
+[#loadModels]
+=== loadModels
+
+It can be used to eagerly load model data, avoiding a slow first query.
```
Comment (languitar) [inline docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc:49]: Maybe one additional sentence helps here:

```suggestion
Returns after trying to eagerly cache a set of specified modules.
This check can be used to avoiding a slow first ModelQl query after launching an MPS instance running this plugin.
```

**Review by languitar — APPROVED** [review]

**Pull request merged**